index.html

```diff
@@ -2,12 +2,75 @@
 <html>
 <head>
 <link href="style.css" rel="stylesheet" type="text/css">
+<script src="https://kit.fontawesome.com/173db1d6d6.js" crossorigin="anonymous"></script>
 <title>Portafolio</title>
 </head>
 <body>
 
-<h1>This is a Heading</h1>
-<p>This is a paragraph.</p>
+    
+<div class="wrapper">
+
+    <nav class="toolbar">
+        <ul>
+            <li class="item">Edahí Gutiérrez </li>        
+            <li id="item-right" class="item">&#9776;</li>                    
+        </ul>        
+    </nav>
+
+    <h1>Hey There. <br>
+        I’m Edahí</h1>
+
+    <h2 id="intro">I’am a Software Developer</h2>   
```

Comment:
- To help improve the HTML semantics on your site, please use a `p` tag instead. 🙏

Reply:
Here I prefer to keep <h1> and <h2> because it gives the correct size for the fonts.